### DISCUSSION TITLE
"Cannot print warningString, please rebuild Ammo.js using 'debug' flag debugDrawer.js:145"

### DISCUSSION THREAD

**Discussion by Lyoko-Jeremie** [enable3d/enable3d#156] [Help / Questions] [answered]
i got above error when set NaN to `setVelocity` , how about it ?
why not print the `warningString` ? think the `warningString` will help some one in some times.

**Comment by yandeu** [comment, marked as answer]
Ammo.js is written in C++ and is therefore very strict. If it wants a number, it needs a number.

But you can use `setVelocityX()`, `setVelocityY()` and `setVelocityZ()` instead.

See [`physicsBody.ts`](https://github.com/enable3d/enable3d/blob/master/packages/common/src/physicsBody.ts).
